src/python/ImpactX.cpp

```diff
@@ -53,6 +54,85 @@ void init_ImpactX(py::module& m)
                 amrex::ParmParse::addfile(filename);
             })
 
+        .def_property("n_cell",
+            [](ImpactX & /* ix */) {
+                std::vector<int> n_cell;
+                amrex::ParmParse pp_amr("amr");
+                pp_amr.getarr("n_cell", n_cell);
+                return n_cell;
+            },
+            [](ImpactX & /* ix */, std::array<int, AMREX_SPACEDIM> /* n_cell */) {
+                throw std::runtime_error("setting n_cell is not yet implemented");
+                /*
+                amrex::ParmParse pp_amr("amr");
+                amrex::Vector<int> const n_cell_v(n_cell.begin(), n_cell.end());
+                pp_amr.addarr("n_cell", n_cell_v);
+
+                int const max_level = ix.maxLevel();
+                for (int lev=0; lev<=max_level; lev++) {
+                    ix.ClearLevel(lev);
+                    // TODO: more...
+                }
```

Comment:
Question for @WeiqunZhang how to change number of cells in an `AmrCore` object at runtime :)

Reply:
From what we chatted to far on Slack, we need to update the data members of AmrCore, AmrMesh, and AmrParGDB.
All MultiFabs need to be updated separately and are not automatically changed. Not sure what we need to update in particle containers, e.g., `parGDB` related.